### PR TITLE
test:refactor update URL, pages and tests

### PR DESCRIPTION
**Pull request by lufernando100**
Tests were updated, regarding the new workshop.

### CODE REVIEW

File: cypress/page/item-page.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class ItemPage {
+  private itemName: string;
+
+  constructor() {
+    this.itemName = "#add-to-cart-sauce-labs-bolt-t-shirt"
+  }
+  public selectItem(): void {
+    cy.wait(500)
```
Comment (danielgalvis98) [inline cypress/page/item-page.ts]: It is not recommended to add this kind of waits

File: cypress/page/login-page.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    private loginURL: string;
+    private userNameField: string;
+    private passwordField: string;
+    private loginButton: string;
+  
+    constructor() {
+      this.loginURL = "https://www.saucedemo.com/";
+      this.userNameField = ".login-box > form > div > input#user-name";
+      this.passwordField = ".login-box > form > div > input#password";
+      this.loginButton = "input[type='submit']";
```
Comment (danielgalvis98) [inline cypress/page/login-page.ts]: Is there a way to improve these selectors?

File: cypress/page/menu-content-page.ts [hunk 3]
```diff
@@ -1,25 +1,39 @@
...
   constructor() {
-    this.tShirtMenu = "li:nth-child(3) [title='T-shirts']";
-    this.menuContentPageURL = "http://automationpractice.com/";
-    this.dressMenu = "#block_top_menu > ul > li:nth-child(2) > a";
+    this.menuContentPageURL = "https://www.saucedemo.com/";
   }
 
   public visitMenuContentPage(): void {
     cy.visit(this.menuContentPageURL);
+    cy.get("input[id='user-name']").type("standard_user"); //(2)
```
Comment (danielgalvis98) [inline cypress/page/menu-content-page.ts:10]: It would be good to the selectors as variables

File: cypress/page/products-list-page.ts [hunk 4]
```diff
@@ -1,26 +1,11 @@
...
   private productList: string;
-  private proceedToCheckoutButton: string;
-  private productos: string;
 
   constructor() {
-    this.productList = "a[title='Add to cart']";
-    this.proceedToCheckoutButton =
-      ".button-container a[title='Proceed to checkout']";
-    this.productos = ".product-container";
+    this.productList = "#item_1_title_link > .inventory_item_name";
```
Comment (danielgalvis98) [inline cypress/page/products-list-page.ts:5]: Is there a way to avoid querying by class?

File: cypress/page/item-page.ts (hunk 1 above)
Comment (lufernando100) [reply cypress/page/item-page.ts]: It was deleted. I forgot that for a test

File: cypress/page/menu-content-page.ts (hunk 3 above)
Comment (lufernando100) [reply cypress/page/menu-content-page.ts:10]: Menu-Content-page file was deleted

File: cypress/page/login-page.ts (hunk 2 above)
Comment (lufernando100) [reply cypress/page/login-page.ts]: Those locators were updated by ID

File: cypress/page/products-list-page.ts (hunk 4 above)
Comment (lufernando100) [reply cypress/page/products-list-page.ts:5]: It will be changed in the next steps

**Review by danielgalvis98 — APPROVED** [review]

**Pull request merged**